gradle.properties

```diff
@@ -8,4 +8,4 @@ appDisplayName = Security.txt
 vendorName = Bouvet Norge AS
 vendorUrl = https://www.bouvet.no
 xpVersion = 7.0.0
```

Comment:
Testing on a 7.0.0 distro is difficult, so I think we should bump this to a currently available version of XP. At the time of writing, this is 7.10.0.

Reply:
Bumped version to 7.10.0, everything still seems to be working as expected.